src/test/java/org/broadinstitute/hellbender/utils/hdf5/HDF5LibraryUnitTest.java

```diff
@@ -43,32 +46,24 @@
     @SuppressWarnings("FieldCanBeLocal")
     private static double TEST_PON_VERSION = 1.3;
 
-    @Test(groups = "supported")
+    @BeforeClass
     public void testIsSupported() {
-        Assert.assertTrue(HDF5Library.isSupported());
+        Assert.assertTrue(new HDF5Library().load(null));
```

Comment:
probably not. The reason is that we have no java equivalent for hdf5. So I'd prefer to know right away when stuff stops working on travis rather than have it skip silently.